pom.xml

```diff
@@ -82,6 +82,24 @@
          </configuration>
        </plugin>
      </plugins>
+    <pluginManagement>
+      <plugins>
+      <plugin>
+        <groupId>org.jenkins-ci.tools</groupId>
+        <artifactId>maven-hpi-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <archive>
+            <index>true</index>
+            <manifestEntries>
+              <Hudson-Version>1.609.18.1</Hudson-Version>
+              <Jenkins-Version>1.609.18.1</Jenkins-Version>
```

Comment:
Mmm…rather dangerous. It would be much too easy to inadvertently introduce a dependency on some unrelated API added between 1.609 and 1.651.

(In https://github.com/jenkinsci/pipeline-plugin/pull/197 I used this trick only for a baseline swap _within_ an LTS line, where the risk of such an added API was quite low.)